.speakeasy/gen.lock

```diff
@@ -3,21 +3,21 @@ id: 463716ea-2172-44e6-ac63-a7a5b29ba606
 management:
   docChecksum: 23afa0eb52b8b4d89afaf090600ec479
   docVersion: "1.0"
-  speakeasyVersion: 1.465.1
-  generationVersion: 2.488.4
+  speakeasyVersion: 1.468.1
+  generationVersion: 2.492.3
   releaseVersion: 0.4.0
```

Comment:
please bump the release version to 0.5.0

Reply:
you can set it manually in this file and then run `speakeasy run --skip-version` again